fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/event/election/fragments/ElectionFragment.java

```diff
@@ -149,14 +150,27 @@ public View onCreateView(
                   "User should not be able to use the action button in this state :" + state);
           }
         });
-
-    election.getState().observe(getViewLifecycleOwner(), eventState -> setupElectionContent());
+    laoDetailViewModel
+        .getCurrentLao()
+        .observe(
+            getActivity(),
+            laoView -> {
+              Log.d(TAG, "lao updated");
+              setupElectionContent();
+            });
 
     return view;
   }
 
   private void setupElectionContent() {
-    EventState electionState = election.getState().getValue();
+    if (getContext() == null) {
+      return;
+    }
```

Comment:
I am not sure I understand why this is needed

Reply:
Because for now this will get called everytime the LAO is updated which may occur when the fragment is not displayed and the context is null. 
I had no better idea how to handle this before the Lao object is split

Reply:
Please add a comment so we still understand the reasoning in many months...